apps/web/github.talon

```diff
@@ -7,51 +7,87 @@ win.title: /github.com/
 focus search: key(s)
 go to notifications: insert("gn")
 go to dashboard: insert("gd")
-show keyboard shortcuts: key(?)
+keyboard shortcuts show: key(?)
+selection move down: key(j)
+selection move up: key(k)
+selection toggle: key(x)
+selection open: key(o)
+# legacy [verb noun] commands to be removed at a later time
 move selection down: key(j)
 move selection up: key(k)
 toggle selection: key(x)
 open selection: key(o)
+show keyboard shortcuts: key(?)
 
 # repositories
 go to code: insert("gc")
 go to issues: insert("gi")
 go to pull requests: insert("gp")
 go to wiki: insert("gw")
-
-# network graph
+go to actions: insert("ga")
+go to projects: insert("gb")
+go to discussions: insert("gg")
 
 # source code browsing
-find file: key(t)
+file find: key(t)
 jump to line: key(l)
-switch (branch|tag): key(w)
-expand url: key(y)
+(branch|tag) switch: key(w)
+url expand: key(y)
 (show|hide) [all] in line notes: key(i)
+blame [view] open: key(b)
```

Comment:
I'd lean towards "show"? Also skeptical of the optional "view"

Reply:
View is probably needs to be not optional.
Terminology/phrase is used from official github shortcuts help page https://docs.github.com/en/get-started/using-github/keyboard-shortcuts#source-code-browsing.
Didn't want to invent something new, not how it was described in doc.

Reply:
Set `blame view open` according to phrase is used to describe this shortcut 
> Open blame view. For more information, see "Tracing changes in a file."

Reply:
@pokey Could you check now, please?